internal/storage/index/builder.go

```diff
@@ -94,6 +98,9 @@ func build(ctx context.Context, fsys fs.FS, opts buildOptions) (Index, error) {
 				d.Name() == util.TestDataDirectory ||
 				util.IsHidden(d.Name()) {
 				return fs.SkipDir
+			} else if strings.HasSuffix(filePath, schema.Directory) {
```

Comment:
Why `HasSuffix`? Wouldn't that potentially have false positives like `foo/bar_schemas`? What about 
```suggestion
			} else if d.Name() == schema.Directory {
```